Assets/Plugins/FMOD/Resources/FMODStudioSettings.asset

```diff
@@ -390,17 +390,17 @@ MonoBehaviour:
   HideSetupWizard: 1
```

Comment:
Are these FMOD changes intentional?

Reply:
no I don't think I touched anything in fmod. But I didn't know if undoing this would break audio & therefore behavior on the enemy prefabs that I updated so I left it in

Reply:
Anything in Assets/Plugins can safely be left alone. If possible can these changes be removed? E.g. if these changes were added in a separate commit?

No worries if not!

Reply:
ahhh sorry.. it's all in the same commit

Reply:
it looks like it just altered the order of a bunch of list elements without actually changing anything

Reply:
no worries, worst case is that I can pull down and change on my end if needed. may not even need to 🤞